readthedocs/oauth/services/github.py

```diff
@@ -515,6 +516,10 @@ def send_build_status(self, build, commit, state):
         # Catch exceptions with request or deserializing JSON
         except (RequestException, ValueError):
             log.exception('GitHub commit status creation failed for project.')
+        except CustomOAuth2Error as e:
+            if e.error != "invalid_grant":
+                raise
+            log.info("Invalid GitHub grant for user.")
```

Comment:
```suggestion
            log.info("Invalid GitHub grant for user.", exc_info=True)
```

Same for the other one